pkg/AnewLibrary/Locale/Atoms/Weather.js

```diff
@@ -0,0 +1,112 @@
+export const atom = (log, resolve) => ({
+/**
+ * @license
+ * Copyright 2023 Atom54 LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+async update({geolocation}, state, {service, isDirty}) {
+  const latitude = geolocation?.latitude?.toFixed(2);
+  const longitude = geolocation?.longitude?.toFixed(2);
+  if (latitude && longitude && isDirty('geolocation')) {
+    const weather = await service('WeatherApiService', 'weatherData', geolocation);
+    log(weather);
+    state.weather = weather;
+    return {weather};
+  }
+},
+
+render(inputs, {weather}) {
+  if (weather) {
```

Comment:
probably should use `shouldRender` instead.

Reply:
done